distribution/docs/src/main/resources/content/_securing/hardening-checklist.adoc

```diff
@@ -9,20 +9,20 @@
 
 The following list enumerates the required mitigations needed for hardening:
 
+* [ ] <<_auditing,Configure Auditing>>
 * [ ] <<_setting_directory_permissions, Set Directory Permissions>>
 * [ ] <<_managing_keystores_and_certificates,Configure Keystore and Certificates>>
-* [ ] <<_managing_certificate_revocation_list_crl,Configure Certificate Revocation List>>
 * [ ] <<_disallowing_login_without_certificates,Disallow Login Without Certificates>>
-* [ ] <<_restricting_access_to_admin_console, Restricting Access to ${admin-console}>>
-* [ ] <<_restricting_feature_app_service_and_configuration_access, Restrict Feature, App, Service, and Configuration Access>>
-* [ ] <<_limiting_access_to_the_sts, Limit Access to the STS>>
-* [ ] <<_removing_default_users,Remove Default Users>>
-* [ ] <<_configuring_guest_access,Configure Guest User Authentication>>
+* [ ] <<_managing_certificate_revocation_list_crl,Configure Certificate Revocation List>>
 ** [ ] <<_denying_guest_user_access,Deny Guest User Access>> (if denying Guest users)
 ** [ ] <<_allowing_guest_user_access,Allow Guest User Access>> (if allowing Guest users)
 * [ ] <<_configuring_guest_claim_attributes,Configure Guest Claim Attributes>> (if allowing Guest users)
+* [ ] <<_configuring_guest_access,Configure Guest User Authentication>>
+* [ ] <<_limiting_access_to_the_sts, Limit Access to the STS>>
+* [ ] <<_restricting_access_to_admin_console, Restricting Access to ${admin-console}>>
+* [ ] <<_restricting_feature_app_service_and_configuration_access, Restrict Feature, App, Service, and Configuration Access>>
+* [ ] <<_removing_default_users,Remove Default Users>>
 * [ ] <<_hardening_solr_index,Harden Solr Index>>
-* [ ] <<_isolating_solr_cloud_and_zookeeper,Isolate Solr Cloud and Zookeeper>>. (If using)
-* [ ] <<_auditing,Configure Auditing>>
 * [ ] <<_environment_hardening,Environment Hardening>>
+* [ ] <<_isolating_solr_cloud_and_zookeeper,Isolate Solr Cloud and Zookeeper>>. (If using)
```

Comment:
❓ Was there a particular reason for reordering this list?

Reply:
Now the list is parallel to the order of the sections themselves